packages/frontend/component/src/ui/modal/confirm-modal.tsx

```diff
@@ -10,7 +10,7 @@ import { Modal } from './modal';
 import * as styles from './styles.css';
 
 export interface ConfirmModalProps extends ModalProps {
-  confirmButtonOptions?: ButtonProps;
+  confirmButtonOptions?: ButtonProps & { autoFocus?: boolean }; // Add autoFocus here
```

Comment:
The ﻿autoFocus property already exists on ﻿ButtonProps, so the changes made in this PR are unnecessary. Furthermore, I haven't seen any use of this specified property in this PR.